openfisca_france/parameters/prelevements_sociaux/pss/plafond_securite_sociale_mensuel.yaml

```
@@ -0,0 +1,487 @@
+    2005-01-01: 2004-11-28
+    2004-01-01: 2003-12-06
+    2003-01-01: 2002-11-23
+    2002-01-01: 2001-11-17
+    2001-01-01: 2000-12-29
+    2000-01-01: 1999-12-10
+    1999-01-01: 1998-12-30
+    1998-01-01: 1997-12-30
+    1997-01-01: 1996-12-29
+    1996-07-01: 1995-12-31
+    1996-01-01: 1995-12-31
+    1995-07-01: 1994-12-29
+    1995-01-01: 1994-12-29
+    1994-07-01: 1993-12-17
+    1994-01-01: 1993-12-17
+    1993-07-01: 1992-12-30
+    1993-01-01: 1992-12-30
+    1992-07-01: 1991-12-31
+    1992-01-01: 1991-12-31
+    1991-07-01: 1991-01-01
+    1991-01-01: 1991-01-01
+    1990-07-01: 1999-01-03
+    1990-01-01: 1990-01-03
+    1989-07-01: 1989-07-01
+    1989-01-01: 1988-12-31
+    1988-07-01: 1987-12-30
+    1988-01-01: 1987-12-30
+    1987-07-01: 1987-01-01
+    1987-01-01: 1987-01-01
+    1986-07-01: 1986-01-01
+    1986-01-01: 1986-01-01
+    1985-07-01: 1984-12-29
+    1985-01-01: 1984-12-29
+    1984-07-01: 1984-06-30
+    1984-01-01: 1983-12-31
+    1983-07-01: 1983-06-29
+    1983-01-01: 1982-12-29
+    1982-07-01: 1982-06-30
+    1982-01-01: 1981-12-31
+    1981-01-01: 1980-12-28
+    1980-01-01: 1979-12-29
+    1979-01-01: 1978-12-28
+    1978-01-01: 1977-12-28
+    1977-01-01: 1976-12-30
+    1976-01-01: 1975-12-30
+    1975-01-01: 1974-12-31
+    1974-01-01: 1973-12-30
+    1973-01-01: 1972-12-30
+    1972-01-01: 1971-12-31
+    1971-01-01: 1970-12-23
+    1970-01-01: 1969-12-31
+    1969-01-01: 1968-12-31
+    1968-01-01: 1967-12-30
+    1967-01-01: 1966-12-28
+    1966-01-01: 1965-12-30
+    1965-01-01: 1964-12-27
+    1964-01-01: 1963-12-29
+    1963-01-01: 1962-12-27
+    1962-01-01: 1961-12-30
+    1961-04-01: 1961-02-18
+    1961-01-01: 1960-06-30
+    1960-07-01: 1960-06-30
+    1960-01-01: 1959-01-03
+    1959-01-01: 1959-01-03
+    1958-01-01: 1957-12-28
+    1955-10-01: 1955-09-30
+    1952-04-01: 1952-04-15
+    1951-10-01: 1951-09-27
+    1951-01-01: 1950-12-31
+    1949-03-01: 1949-02-25
+    1948-03-01: 1948-03-03
+    1947-10-01: 1947-09-25
+    1946-10-01: 1946-10-08
+    1946-07-01: 1945-10-06
+    1938-07-01: 1938-06-15
+  notes:
+    1997-01-01: Depuis 1997, revalorisation chaque début d'année.
+    1996-07-01: Les plafonds annuels sont inexacts sur legislation.cnav.fr (12*montant mensuel) au lieu de la somme des plafonds mensuels sur l'année.
+    1982-01-01: Entre 1982 et 1997, revalorisation en janvier et juillet (cf. règles fixées par le décret 82-542 du 29/06/1982).
+    1962-01-01: Entre 1962 et 1982, revalorisation en janvier uniquement.
+    1960-01-01: Changement des anciens francs aux nouveaux francs
+    1951-10-01: Référence erronée dans legislation.cnav.fr
+    1938-07-01: "JO sur gallica: http://gallica.bnf.fr/ark:/12148/bpt6k6550526r/f20.image.r=journal%20officiel%201948.langFR"
+    1937-01-01: Voir référence sur Gallica.bnf.fr
+    1936-01-01: Voir référence sur Gallica.bnf.fr
+    1930-07-01: Voir référence sur Gallica.bnf.fr
+  description_en: Social security ceiling (1930-2016)
```

Je ne crois pas qu'il y ait une date de fin
```suggestion
  description_en: Social security ceiling
```